internal/manifest/validate_kubernetes.go

```diff
@@ -41,16 +41,27 @@ func (k *Kubernetes) Validate(m *Manifest) error {
 		}
 		names[cluster.Name] = true
 
+		// check for re-use of the same nodepool
+		computeNames := make(map[string]bool)
+		controlNames := make(map[string]bool)
 		for _, pool := range cluster.Pools.Control {
 			if !m.nodePoolDefined(pool) {
 				return fmt.Errorf("control nodepool %q used inside cluster %q not defined inside manifest", pool, cluster.Name)
 			}
+			if _, ok := controlNames[pool]; ok {
+				return fmt.Errorf("nodepool %q used multiple times as control nodepool, this effect can be achieved by increasing the \"count\" field or adjusting the \"autoscaler\" field", pool)
```

Comment:
Maybe we can add something along lines `nodepool %q used multiple times as control nodepool, this effect can be achieved by increasing the \"count\" field, adjusting the \"autoscaler\ field or defining a new nodepool with a different name`

Applies for the rest of these messages